### DISCUSSION TITLE
Incredible: It works!

### DISCUSSION THREAD

**Discussion by jojo2234** [Path-Check/paper-cred-demo#10] [Q&A]
Hello, I have made a project https://github.com/jojo2234/GreenPass-Experiments similar and I need a Private Key to sign the document. How do you have got one? Anyway are these keys encrypted? I have tested with my App the output of your QRCode and I get a working result the App read the QR Code and say not valid but show Name and Surname and Born Date and the only reason was due to an expired date. So your project works because you have got the Private Key :D. I want to make my project works too, could you give me a bit of help? Could I use your keys? Thank you.

**Comment by vitorpamplona** [comment]
Hi Alessandro, 

We don't have a production key for EU certificates. Only member states have them.

**Reply by jojo2234** [reply]
I don't understand. Why I get a working result with the QR Code generated with your app? The QR Code is recognized, name, surname and born date are shown in the App. Your signature must be correct. Is there a demo signature in the DB of the App?

**Reply by vitorpamplona** [reply]
Which app recognized the QR?

**Reply by jojo2234** [reply]
No sorry, :) I was wrong. I have just realized that the app can show name and surname of an invalid certificate. But when I generate a QR code with my script I was used to get an unreadable certificate and when I see on your QR Code the name I was thinking that this difference was due to a signature instead was due to an invisible character (space at the end of the string to querify) in my qrcodes. :D Now even in mine it's possible see the name. https://github.com/jojo2234/GreenPass-Experiments/blob/main/NON_VALIDO.jpg

**Comment by Pr0metheus2** [comment]
Hello jojo2234, 
your repository has been disabled by GitHub, do you have any other location where I can download it, please?
Thank you.
